schema/crdb/add-instance-id-to-migrations/up02.sql

```diff
@@ -0,0 +1,45 @@
+-- A table of the states of current migrations.
+CREATE TABLE IF NOT EXISTS omicron.public.migration (
+    id UUID PRIMARY KEY,
+
+    /* The ID of the instance that was migrated */
+    instance_id UUID NOT NULL,
+
+    /* The time this migration record was created. */
+    time_created TIMESTAMPTZ NOT NULL,
+
+    /* The time this migration record was deleted. */
+    time_deleted TIMESTAMPTZ,
```

Comment:
Does this need a modification time?

Reply:
Ah, I think this is covered by the two "updated" time fields below, is that right?

Reply:
Yeah, that's correct; once a migration record is inserted, updates will touch either the source OR target fields. I should probably add a comment about this.

Reply:
Added a comment explaining this in 3ec590483b33943bf0eb821a3ec0069ad77e6c30